cmd/link/ocmrole/cmd.go

```diff
@@ -133,9 +133,16 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 	linked, err := r.OCMClient.LinkOrgToRole(orgAccount, roleArn)
 	if err != nil {
 		if errors.GetType(err) == errors.Forbidden || strings.Contains(err.Error(), "ACCT-MGMT-11") {
-			r.Reporter.Errorf("Only organization admin can run this command. "+
+			ocmAccount, localErr := r.OCMClient.GetCurrentAccount()
+			if localErr != nil {
+				r.Reporter.Errorf("Error getting OCM account: %v", localErr)
+				return localErr
+			}
+
+			r.Reporter.Errorf("OCM account '%s' has no permission for this command, "+
```

Comment:
We use the term `current account` in other places in the error message, I wonder if it's worth changing all of them to `OCM account`?
Not sure if the user is familiar with the term OCM, on the other hand, `current account` can be confusing with the current AWS account.

Reply:
@oriAdler Just wondering if the output is easier to understand if we display the username instead of the account id? e.g I feel this is more readable:

```
Your account 'rblake.openshift' has no permission for this command. Only an organization admin can run this command. Please ask an organization admin to run the following command:
```

Reply:
I agree that current account can be confused with user aws account user.
Can we word it along the lines of, "OCM account of the user" so that it is more descriptive to the user?

Reply:
> We use the term `current account` in other places in the error message, I wonder if it's worth changing all of them to `OCM account`? Not sure if the user is familiar with the term OCM, on the other hand, `current account` can be confusing with the current AWS account.

Red Hat Account maybe 🤔 ?

Reply:
Changed to `Red Hat account` with the user name included, thanks.